src/reference/asciidoc/redis.adoc

```diff
@@ -769,6 +769,94 @@ IMPORTANT: The `task-executor` has to be configured with more than one thread fo
 The `errorChannel` can be used to process those errors, to avoid restarts, but it preferable to not expose your application to the possible deadlock situation.
 See Spring Framework https://docs.spring.io/spring/docs/current/spring-framework-reference/integration.html#scheduling-task-executor-types[Reference Manual] for possible `TaskExecutor` implementations.
 
+[[redis-stream-outbound]]
+=== Redis Stream Outbound Channel Adapter
+
+Spring Integration 5.4 introduced Reactive Redis Stream outbound channel adapter to write Message payload into Redis stream.
+Outbound Channel adapter uses `ReactiveStreamOperations.add(...)` to send `Message` to stream.
+The following example shows how to use Java Configuration and Service class for Redis Stream Outbound Channel Adapter.
+
+====
+[source,java]
+----
+@Component
+public class MyClass {
```

Comment:
As I said before: we don't need the whole class for configuration.
A code snippet starting with `@ServiceActivator` is fully enough in this context.